packages/code-du-travail-frontend/src/api/modules/legal-articles/service.ts

```diff
@@ -0,0 +1,33 @@
+import { elasticDocumentsIndex, elasticsearchClient } from "../../utils";
+import { getLegalArticle } from "./queries";
+import { DocumentElasticResult, ElasticLaborCodeArticle } from "./type";
+
+// TODO bouger dans un fichier common ??
+export const getItemBySlug = async <
+  V,
+  K extends keyof DocumentElasticResult<V>
+>(
+  fields: K[],
+  body: any
```

Comment:
J'ai trouvé le type ! 

```suggestion
  body: SearchRequest
```

Et l'import : 
```
import { SearchRequest } from "@elastic/elasticsearch/lib/api/types";
```

Reply:
top ! merci :pray: